client/src/components/Login.js

```diff
@@ -0,0 +1,17 @@
+import styled from '@emotion/styled';
+
+const LoginName = styled.input`
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  text-align: center;
+  font-size: 12px;
+  color: ${props => props.theme.secondary};
+  border: 1px solid ${props => props.theme.third};
```

Comment:
You could name it "props.theme.tertiary"

Reply:
Thanks, good suggestion.